src/App.tsx

```diff
@@ -2,13 +2,22 @@ import React from 'react';
 
 import './App.scss';
 
-// import posts from './api/posts';
-// import comments from './api/comments';
-// import users from './api/users';
+import posts from './api/posts';
+import comments from './api/comments';
+import users from './api/users';
+
+import { PostList } from './components/PostList/PostList';
+
+const preparedPosts = posts.map(post => ({
+  ...post,
+  user: users.find(user => user.id === post.userId),
+  comment: comments.find(comment => comment.id === post.id),
```

Comment:
there're should be multiple comments for one post (array)

```suggestion
  comments: comments.filter(comment => comment.id === post.id),
```